tango/integrations/torch/train_callback.py

```diff
@@ -55,6 +55,13 @@ def __init__(
         self.validation_dataloader = validation_dataloader
         self.lr_scheduler = lr_scheduler
 
+    @property
+    def step_id(self) -> str:
+        """
+        The unique ID of the current :class:`~tango.Step`.
+        """
+        return self.train_config.step_id
```

Comment:
I don't have the overview here, but will this work if we don't use configuration files, and thus we don't have a config?

Reply:
Yea, the `train_config` object is not a  configuration file, it's just an internal object we pass around here.